01_Core/week04/ecommerce/script.js

```diff
@@ -0,0 +1,68 @@
+var slidesArray = [];
+var slideNumber = 0;
+
+var mainSlide0 = document.querySelector(".slide-1");
+var mainSlide1 = document.querySelector(".slide-2");
+var mainSlide2 = document.querySelector(".slide-3");
+slidesArray[0] = mainSlide0;
+slidesArray[1] = mainSlide1;
+slidesArray[2] = mainSlide2;
+
+
+
+/************************************Function to move slide forward*********************************/ 
+function slideNext () 
+{
+    console.log(slidesArray);
```

Comment:
I would say its best practice to remove `console.log`s you dont need before submitting.
its okay now but just keep it in mind for later.